subsys/bluetooth/controller/ll_sw/ull_iso.c

```diff
@@ -329,7 +329,7 @@ uint8_t ll_setup_iso_path(uint16_t handle, uint8_t path_dir, uint8_t path_id,
 		sync_iso = ull_sync_iso_by_stream_get(stream_handle);
 		lll_iso = &sync_iso->lll;
 
-		role = 1U; /* FIXME: Is this correct role value? */
+		role = BT_ROLE_BROADCAST;
```

Comment:
I'm a bit confused about the changes here: The `if` clause we are in here is for the broadcast sink - Why are we using `BT_ROLE_BROADCAST` for that? Or is `BT_ROLE_BROADCAST` a common value for sink and source? 

If `BT_ROLE_BROADCAST` is only for the sync receiver (broadcast sink), then perhaps it's better named `BT_ROLE_BROADCAST_SINK`?